mp/src/public/vgui_controls/Frame.h

```diff
@@ -133,6 +133,11 @@ class Frame : public EditablePanel
 	// Temporarily enables or disables the fade effect rather than zeroing the fade times as done in DisableFadeEffect
 	void SetFadeEffectDisableOverride( bool disabled );
 
+    void SetInFocusBgColor(Color col);
+    void SetOutOfFocusBgColor(Color col);
+    Color GetInFocusBgColor();
+    Color GetOutOfFocusBgColor();
```

Comment:
Since these are basic getters and setters, they can be defined in the header:
```suggestion
    void SetInFocusBgColor(Color col) { m_InFocusBgColor = col; }
    void SetOutOfFocusBgColor(Color col) { m_OutOfFocusBgColor = col; }
    Color GetInFocusBgColor() const { return m_InFocusBgColor; }
    Color GetOutOfFocusBgColor() const { return m_OutOfFocusBgColor; }
```

Reply:
Actually won't even need these I think, as they are able to be overwritten in a resource control file.